app/__init__.py

```diff
@@ -211,5 +212,6 @@ def index() -> ResponseReturnValue:
     # should make an intentional decision for when to be setting this
     app.config["SEND_FILE_MAX_AGE_DEFAULT"] = 3600
     app.add_template_global(AuthorisationHelper, "authorisation_helper")
+    app.add_template_global(FlashMessageType, "flash_message_type")
```

Comment:
One for us to take away into the future - wrapping enums/types somehow so we're not just bunging them all in a bit randomly.